test.js

```diff
@@ -80,6 +80,72 @@ test('spawn cli and test input flag', async t => {
 	t.is(stdout, 'bar');
 });
 
+test('spawn cli and test required flag with missing args', async t => {
```

Comment:
Can you move these new tests into a new test file? And also move all tests into a `test` directory?